presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -182,7 +197,7 @@ protected void assertAccessDenied(
             fail("Expected " + AccessDeniedException.class.getSimpleName());
         }
         catch (RuntimeException e) {
-            assertExceptionMessage(e, exceptionsMessageRegExp);
+            assertExceptionMessage(e, ".*Access Denied: " + exceptionsMessageRegExp);
```

Comment:
Relates as I use ``assertExceptionMessage`` in ``assertQueryFails`` which I added.